app/globals.css

```
@@ -4,82 +4,259 @@
+    --radius-pill: 999px;
+    --page-gutter: clamp(1.25rem, 2vw, 2rem);
+    --section-space: clamp(4.5rem, 8vw, 5rem);
+    --transition-standard: 220ms cubic-bezier(0.22, 1, 0.36, 1);
+    --transition-slow: 480ms cubic-bezier(0.2, 1, 0.22, 1);
+  }
 
-    --card: 0 0% 100%;
-    --card-foreground: 240 10% 3.9%;
+  * {
+    @apply border-border-subtle;
+    box-sizing: border-box;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)